README.md

```diff
@@ -45,6 +45,13 @@ The functions that are currently available are as follows:
   identifier that can be used to reference this scheduled job (e.g.,
   for cancellation) later. A group identifier `group-id` can be provided that
   allows jobs in the same group to be stopped at the same time.
+* `cron [cron-string f]`: schedules a job that will call `f` in accordance with the
+  cron schedule indicated by the 'cron-string'. Returns an identifier that can 
```

Comment:
I think we need some explanation / description of what a valid cron string is somewhere in this document.  Could point to the quartz documentation somewhere, or could be included in this repo.